web/src/js/index.js

```diff
@@ -231,43 +235,55 @@ function getExecutionLog(executionLogUrl) {
   })
     .then((response) => response.json())
     .then((data) => {
-      app.executionLogTable = data["response"]["body"];
       app.executionLogKey = decodeURIComponent(executionLogUrl);
+      app.executionLogTable = data["response"]["body"];

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792603)
```suggestion
      app.executionLogTable = data["response"]["body"];
```